cobigen-setup/files/cliExecutionCommands.md

```diff
@@ -0,0 +1,23 @@
+CobiGen CLI is installed inside your devonfw distribution. In order to execute it follow the next steps:
+
+1. Now, we have to go into the devonfw installation directory to execute the commands.
+
+`cd devonfw`{{execute T1}}
+
+2. To install cobigen inside devonfw ide execute below command. This command will install cobigen inside ide.

```

Comment:
I don't know why but in Katacoda all numbers are displayed as '1.'. Therefore use bullet points or just use normal text without any listing sign before.
And pay attention on the writing of CobiGen. It should be writed every time the same way.